cpr/error.cpp

```diff
@@ -1,202 +1,165 @@
 #include "cpr/error.h"
-#include <cstdint>
-#include <curl/curl.h>
```

Comment:
I guess you are missing those two includes for clang-tidy to work.

Reply:
Oh, I removed them because there was apparant reason for them. 
I will add them back, but why does clang-tidy need them?